include/mbedtls/error.h

```diff
@@ -50,50 +50,42 @@
  *
  * Module   Nr  Codes assigned
  * MPI       7  0x0002-0x0010
- * GCM       3  0x0012-0x0014   0x0013-0x0013
- * BLOWFISH  3  0x0016-0x0018   0x0017-0x0017
- * THREADING 3  0x001A-0x001E
- * AES       5  0x0020-0x0022   0x0021-0x0025
- * CAMELLIA  3  0x0024-0x0026   0x0027-0x0027
- * XTEA      2  0x0028-0x0028   0x0029-0x0029
+ * GCM       2  0x0012-0x0014
+ * BLOWFISH  2  0x0016-0x0018
+ * THREADING 3  0x001C-0x001E
+ * AES       3  0x0020-0x0022   0x0021-0x0021
```

Comment:
This looks a bit funny. Isn't `0x0021` within the range of `0x0020-0x0022` Why does it need a separate entry?

Reply:
The left is the even error codes, the right are the odd error codes

Reply:
Weird. `THREADING` doesn't do that. `BLOWFISH` doesn't do that.

Reply:
Nevermind. `BLOWFISH` doesn't do this because we deleted `0x0017`. `THREADING` also doesn't have odd error codes.